libr/core/config.c

```diff
@@ -7,6 +7,23 @@
 #define SETPREF(x,y,z) r_config_node_desc(r_config_set(cfg,x,y), z);
 #define SETCB(w,x,y,z) r_config_node_desc(r_config_set_cb(cfg,w,x,y), z);
 
+/*------------------------------------------------------------------------------------------*/
+
+static void r_config_set_sort_column(char *column){
+	if(!strcmpi(column,"name"))
+		r_anal_compare = &compareName;
+	else if(!strcmpi(column,"address"))
+		r_anal_compare = &compareAddress;
+	else if(!strcmpi(column,"type")
+		r_anal_compare = &compareType;
+	else if (!strcmpi(column,"size"))
+		r_anal_compare = &compareSize;
+	else if (!strcmpi(column,"dist"))
+		r_anal_compare = &compareDist;
+}
```

Comment:
i would probably prefer to use a for loop and define an array of name: callback structs